.github/workflows/test-e2e.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           k3d cluster create skr -p 10080:80@loadbalancer -p 10443:443@loadbalancer --k3s-arg '--disable=traefik@server:0'
       - name: Provision KCP cluster
         run: |
-          kyma provision k3d --name=kcp -p 9080:80@loadbalancer -p 9443:443@loadbalancer --ci --registry-port 5111
+          kyma provision k3d --name=kcp -p 9080:80@loadbalancer -p 9443:443@loadbalancer -p 9081:8080@loadbalancer --ci --registry-port 5111
```

Comment:
can you just change this `9080:80` to `9080:8080`? I don't think expose `80` port is needed here.

Reply:
The port 80 expose is needed for the communication between kcp and skr. It's advised as well in the developer docs. I"m not sure why though, since the watcher communicates over tls. but some e2e tests fail if this port is not open